trio/_core/_run.py

```diff
@@ -1356,13 +1446,41 @@ class RunContext(threading.local):
 GLOBAL_RUN_CONTEXT: FinalT = RunContext()
 
 
-@attr.s(frozen=True)
-class _RunStatistics:
-    tasks_living = attr.ib()
-    tasks_runnable = attr.ib()
-    seconds_to_next_deadline = attr.ib()
-    io_statistics = attr.ib()
-    run_sync_soon_queue_size = attr.ib()
+class _IOStats(Protocol):
+    @property
+    def backend(self) -> str:
```

Comment:
change this to be a `Literal` as in #2743 ?

Reply:
Should actually just be able to eliminate this entirely, specify the concrete type depending on the platform.